testsuite/src/main/scala/endpoints/testsuite/BasicAuthTestApi.scala

```diff
@@ -12,4 +13,13 @@ trait BasicAuthTestApi extends algebra.Endpoints with algebra.BasicAuthenticatio
     textResponse
   )
 
+  val protectedEndpointViaBuilder =
+    anEndpoint
+      .withMethod(Get)
+      .withUrl(path / "users")
+      .withTextResponse
+      .withBasicAuth
```

Comment:
The equivalent without builders would be:

~~~ scala
authenticatedEndpoint(Get, path / "users", response = textResponse)
~~~

“authentication for responses” means that responses are “authentication aware”: they will return a `403` in case of an unauthorized user.